### ISSUE TITLE
[request] adding Menu background  music ?

### ISSUE THREAD

**Issue by appletv4** [jasarien/Provenance#479]
For Apple TV would it be possible to have background music when you are in the game selection menu 

Like the nes classic 
https://www.youtube.com/watch?v=zb_YhWzWnHQ
https://www.youtube.com/watch?v=dgpBhguhGuk

I know you can't officially add that to provenance for copyright reasons but how about having an option for people to add their own music 

Is it possible to do it now already ? 

Also an option in provenance settings to disable Apple TV screensaver when the app is open

**Comment by marcelvoss** [comment]
I mean, this is possible but is there really a need for this? Provenance may get a bit too cluttered with features only a very small minority of users will use. New features also means new and more code which also needs to be maintained.

**Comment by appletv4** [comment]
@marcelvoss this will be my nes classic I couldn't get in the stores

**Comment by marcelvoss** [comment]
@appletv4 Yeah, I completely understand that. We'll take a look into it, okay?

**Comment by appletv4** [comment]
@marcelvoss cool, thanks

**Comment by sevdestruct** [comment]
There's actually some sites that have this stuff some with APIs if we wanted it to be more Plex like, but until we have #720 I don't see the point. And even then it's going to be a low priority, though cool.

**Comment by JoeMatt** [comment]
#2654 related
